x/crconsent/client/cli/txRequest.go

```diff
@@ -0,0 +1,152 @@
+package cli
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	"github.com/metabelarus/mbcorecr/x/crconsent/types"
+)
+
+func CmdCreateRequest() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create-request [initiator] [recipient] [requestType] [status] [value] [memo] [promoUrl] [creationDt] [finalDt]",
+		Short: "Creates a new request",
+		Args:  cobra.ExactArgs(9),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			argsInitiator := string(args[0])
+			argsRecipient := string(args[1])
+			argsRequestType := string(args[2])
+			argsStatus := string(args[3])
+			argsValue, _ := strconv.ParseInt(args[4], 10, 64)
+			argsMemo := string(args[5])
+			argsPromoUrl := string(args[6])
+			argsCreationDt := string(args[7])
```

Comment:
Most of date parameters should be generated here as it is, not be taken from arguments. It simplifies the interface.
Check NewMsgRequestAuth - as example